.travis.yml

```diff
@@ -32,3 +32,8 @@ script:
   # push docker images to registry if master branch:
   - make push -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master"
   - make push -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master" -e IMAGE_PREFIX="docker.io/avenga"
+  - > 
+    if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then 
+    make test-all build-all push-all -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master";
```

Comment:
The order is wrong: First build, then test and push at last.

Reply:
🤦 Thanks!